builds/e2e/templates/connectivity-deploy.yaml

```diff
@@ -73,6 +77,14 @@ steps:
           -edgeletBranchName "${{ parameters['edgelet.branchName'] }}" \
           -hostPlatform "${{ parameters['metricsCollector.hostPlatform'] }}" \
           -storageAccountConnectionString "${{ parameters['testResultCoordinator.storageAccountConnectionString'] }}" \
+          -devOpsAccessToken $(System.AccessToken) \
+          -devOpsBuildId $(Build.BuildId) \
```

Comment:
This field already exists, right? See testBuildNumber on line 75.

Reply:
Build number e.g. 20200201.3 is different from build Id, e.g. 11263738.